docs/conf/prague/2018/news/scheduling-inclusivity.rst

```diff
@@ -0,0 +1,80 @@
+
+The long version
+----------------
+
+The global Write the Docs team supports major conferences in both Portland and Prague, as well as mentoring smaller events throughout the year and maintaining the community infrastructure.
+
+Since we started organizing the Prague conference in 2015, September has been the month of choice, specifically around the middle of the month.
+
+When we scheduled the conference dates for Prague we considered the following key factors:
+
+- September is far enough from the Portland conference (which is held in May each year), which helps the team to load-balance preparation activities for both conferences.
+- August (and the first week of September) conflicts with the school summer holidays, which would exclude attendees who have school-aged children.
+- The only bank holiday in the Czech Republic that month is on September 28, and there are no other public holidays in neighboring European countries.
+- Mid-September doesn't conflict with any major European conferences in the region (primarily PyCons and open-source conferences).
+- The weather from the end of September takes a turn for the worse in Prague (and in most of Europe), and we want attendees to have a good experience of the city and not just the conference venue.
+- The global core team's yearly offsite and retrospective happens immediately after the Prague conference, and we need to allow for enough time before the end of the calendar year to wrap-up and start planning for the following year.
+
+What happened this year
+~~~~~~~~~~~~~~~~~~~~~~~
+
+When we looked at the calendar to choose the dates for the 2018 conference, we saw that the Jewish New Year holidays span most  of September.
+This does not occur every year, but this year it was extremely difficult to decide on dates in September, since no matter which date we chose, it would conflict with one of the holidays.
+
+When we weighed this problem against the key factors that we use to determine the dates each year, we knew that we had to make a decision that would exclude some people from being able to attend.
+We weren't happy about this decision, but if we moved the conference to any other month, or even to the first weekend of September, we would run the risk of excluding different groups of people, or not being able to run the conference at all this year due to organizational meltdowns.
+
+What could we have done better
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Aside from the risks involved with choosing different dates or canceling the Prague conference altogether, there wasn't much we could have done with regards to the scheduling this year.
+What we realized following the feedback from the community, was that we failed to communicate our decision in a transparent and empathetic way at the time of annoucning the dates.
```

Comment:
so if this is the first sentence in the section, I'd just say 'We failed to communicate the context around our decision...'